ports/x265/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "x265",
   "version": "3.5",
-  "port-version": 1,
+  "port-version": 2,
   "description": "x265 is a H.265 / HEVC video encoder application library, designed to encode video or images into an H.265 / HEVC encoded bitstream.",
   "homepage": "https://github.com/videolan/x265",
```

```suggestion
  "homepage": "https://bitbucket.org/multicoreware/x265_git/",
```
The GH page seems pretty old, and the port don't use this source.

---

Pinging @foobra for response.